src/components/utils.ts

```diff
@@ -0,0 +1,11 @@
+export const bytesToSize = (bytes: number): string => {
+  if(bytes === 0) return '0 B';
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB"];
+  const i = parseFloat(Math.floor(Math.log(bytes) / Math.log(1024)).toString());
```

Comment:
This is a cumbersome way to do it. Why would you want to convert integer to string and then back to float, which is not really a float, but a `number` in JS anyway?

Reply:
Was totally unnecesary, have removed the parseFloat and toString.